packages/trpc/server/middlewares/sessionMiddleware.ts

```diff
@@ -102,21 +102,16 @@ export async function getUserFromSession(ctx: TRPCContextInner, session: Maybe<S
   // This helps to prevent reaching the 4MB payload limit by avoiding base64 and instead passing the avatar url
 
   const locale = user?.locale ?? ctx.locale;
-
-  const isOrgAdmin = !!user.profile?.organization?.members.filter(
-    (member) => (member.role === "ADMIN" || member.role === "OWNER") && member.userId === user.id
-  ).length;
+  const isOrgAdmin = await UserRepository.isAdminOrOwnerOfTeam({
+    userId: user.id,
+    teamId: user.profile.organization?.id ?? -1,
+  });
 
   if (isOrgAdmin) {
     logger.debug("User is an org admin", safeStringify({ userId: user.id }));
   } else {
     logger.debug("User is not an org admin", safeStringify({ userId: user.id }));
   }
-  // Want to reduce the amount of data being sent
-  if (isOrgAdmin && user.profile?.organization?.members) {
-    user.profile.organization.members = [];
```

Comment:
the replacement didn't work at all

Reply:
That's because isOrgAdmin was only true for admins, so this replacement may have worked exclusively for admins, but not role MEMBER.

Reply:
But the user I was impersonating was an admin tho.